base/r_main.c

```diff
@@ -790,9 +790,6 @@ void R_SetupFrame(player_t *player)
 		UpdateState |= I_MESSAGES;
 	}
 
-#if defined(__WATCOMC__) && defined(_DOS)
-	destview = destscreen + (viewwindowx>>2) + viewwindowy*80;
```

Comment:
They are 'used' in I_InitDiskFlash() of i_ibm.c in the original dos version, but it is commented out.

Reply:
Indeed, I saw it too but I don't think anyone is ever going to use it. :-P

Reply:
OK, let's keep it. It doesn't have to exist in sdl sources but keep it in other places